source/features/deprioritize-marketplace-link.tsx

```diff
@@ -24,7 +24,8 @@ features.add({
 	description: 'Moves the "Marketplace" link from the black header bar to the profile dropdown.',
 	screenshot: false,
 	exclude: [
-		features.isGist
+		features.isGist,
+		features.isPasswordPage
```

Comment:
https://github.com/sindresorhus/refined-github/issues/2128#issuecomment-499903145

No feature works on this page. This check does not belong here, as mentioned in that thread